cg/meta/transfer/external_data.py

```diff
@@ -165,6 +176,8 @@ def curate_sample_folder(self, cust_name: str, force: bool, sample_folder: Path)
             sample and force
         ):
             sample_folder.rename(customer_folder.joinpath(sample.internal_id))
+        elif sample_folder.name == "nanopore":
```

Comment:
i think this check should happen outside of this function to keep it more clean

Reply:
Do the check just before curate_sample_folder is being called

Reply:
fixed